WTWSMS/localisation/WtWSMS_religions_unused.csv

```diff
@@ -56,8 +56,6 @@ GOD_OMNIPATER;The Omnipater;;;;;;;;;;;x
 GOD_LUXIMATER;The Luximater;;;;;;;;;;;x
 GOD_THE_FROGMEN;The Frogmen of Wasketville Court;;;;;;;;;;;x
 GOD_CONNOR_ANDREW_DAY;Connor Andrew Day;;;;;;;;;;;x
-banditism;Banditism;;;;;;;;;;;x
-banditism_DESC;The highly heretical Bandit sect claims that the almighty Delorius is not a unicorn, but instead a regular horse named Bandit. Definitely an Athiest sect, thanks Communist Obama!;;;;;;;;;;;x
```

Comment:
If that irritates you out of everything, WtWSMS is not your can of works. :stuck_out_tongue: